### PR TITLE
update docs to reflect the changes in the js sdk

### PR DESCRIPTION
**Pull request by karthik2804**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [ ] The `title, `template`, and `date` are all set
- [ ] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [ ] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [ ] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [ ] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [ ] Headings are using Title Case
- [ ] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [ ] Have tested with `npm run test` and resolved all errors
- [ ] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/spin/javascript-components.md [hunk 1]
```diff
@@ -400,6 +400,6 @@ These are some of the suggested libraries that have been tested and confired to
 
 ## Caveats
 
-- All `spinSdk` related functions and methods can be called only inside the `handleRequest` function. This includes the usage of `fetch`. Any attempts to use it outside the function will lead to an error. This is due to Wizer using only Wasmtime to execute the script at build time, which does not include any Spin SDK support.
+- All `spin` related functions and methods (like `Config`, `Redis`, `Mysql`, `Pg`, `Kv` & `Sqlite`)can be called only inside the `handleRequest` function. This includes the usage of `fetch`. Any attempts to use it outside the function will lead to an error. This is due to Wizer using only Wasmtime to execute the script at build time, which does not include any Spin SDK support.
```
Comment (itowlson) [inline content/spin/javascript-components.md:403]:
```suggestion
- All `spin-sdk` related functions and methods (like `Config`, `Redis`, `Mysql`, `Pg`, `Kv` and `Sqlite`) can be called only inside the `handleRequest` function. This includes the usage of `fetch`. Any attempts to use it outside the function will lead to an error. This is due to Wizer using only Wasmtime to execute the script at build time, which does not include any Spin SDK support.
```

(If you prefer "Spin" to "spin-sdk" at the beginning then capital letter and no monospace - "Spin" is the product, `spin` is the command.  Hashtag brand guidelines that I just made up in my head.)

**Review by itowlson — APPROVED** [review]
Small style and punctuation fixes but otherwise LGTM.

Though I do kind of wonder if we should mention anywhere that JS SDK 0.4 and earlier are _completely different_

**Comment by karthik2804** [comment]
I am not sure what approach we want to take here about the v0.4.0 @itowlson .

**Comment by itowlson** [comment]
Okay, I guess leave the 0.4 problem for now.  I can drop something into the language guide as a separate PR.

**Pull request merged**